packages/matchbox/src/components/Checkbox/styles.js

```diff
@@ -0,0 +1,66 @@
+import { tokens } from '@sparkpost/design-tokens';
+import { Box } from '../Box';
+import { visuallyHidden } from '../../styles/helpers';
+import styled from 'styled-components';
+import { margin } from 'styled-system';
+
+export const Wrapper = styled('div')`
+  ${margin}
+`;
+
+export const StyledBox = styled(Box)`
+  border: 2px solid ${props => (props.error ? tokens.color_red_700 : tokens.color_gray_500)};
```

Comment:
Don't think this gray border is accessible, same with some of the other grays in the disabled styles – will ask UX team to check it out